src/main/resources/citizen/templates/document/claimantDirectionsQuestionnaire.html

```diff
@@ -257,121 +261,160 @@ <h2 class="heading-medium">
-  </tr>
-  {% endfor %}
-  <tr>
-    <td>
-      <div>
-        Do you want the court's permission to hire an expert?
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.courtPermissionForExpertReport }}
-      </div>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <div>
-        Does the claim involve something an expert can still examine?
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.expertExamineNeeded }}
-      </div>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <div>
-        What is there to examine?
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.whatToExamine }}
-      </div>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <div>
-        Briefly explain why you believe an expert is needed
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.reasonWhyExpertAdvice }}
-      </div>
-    </td>
-  </tr>
+  {% if hearingContent.hasExpertReport equals "Yes" %}
+    <tr>
+      <td>
+        Have you already got a report written by an expert?
+      </td>
+      <td>
+        <div>
+          {{ hearingContent.hasExpertReport }}
+        </div>
+      </td>
+    </tr>
+    {% for report in hearingContent.expertReports %}
+    <tr>
+      <td>
+        <div>
+          Report {{ loop.index + 1 }}
+        </div>
+      </td>
+      <td>
+        <div>
+          {{ report.expertName }} <br/>
+          {{ report.expertReportDate }}
+        </div>
+      </td>
+    </tr>
+    {% endfor %}
+
+    {% if hearingContent.courtPermissionForExpertReport is not empty %}
+      <tr>
+        <td>
+          <div>
+            Do you want the court's permission to hire an expert?
+          </div>
+        </td>
+        <td>
+          <div>
+            {{ hearingContent.courtPermissionForExpertReport }}
+          </div>
+        </td>
+      </tr>
+    {% endif %}
+
+    {% if hearingContent.expertExamineNeeded is not empty %}
+      <tr>
+        <td>
+          <div>
+            Does the claim involve something an expert can still examine?
+          </div>
+        </td>
+        <td>
+          <div>
+            {{ hearingContent.expertExamineNeeded }}
+          </div>
+        </td>
+      </tr>
+    {% endif %}
+
+    {% if hearingContent.whatToExamine is not empty %}
+      <tr>
+        <td>
+          <div>
+            What is there to examine?
+          </div>
+        </td>
+        <td>
+          <div>
+            {{ hearingContent.whatToExamine }}
+          </div>
+        </td>
+      </tr>
+    {% endif %}
+
+    {% if hearingContent.reasonWhyExpertAdvice is not empty %}
+      <tr>
+        <td>
+          <div>
+            Briefly explain why you believe an expert is needed
+          </div>
+        </td>
+        <td>
+          <div>
+            {{ hearingContent.reasonWhyExpertAdvice }}
+          </div>
+        </td>
+      </tr>
+    {% endif %}
+
+  {% else %}
+
+    {% if hearingContent.hasExpertReport is not empty %}
```

Comment:
Please remove the if statement here as it's not needed

Reply:
done